configs/config.json

```diff
@@ -9,5 +9,10 @@
     "AptreeListPage": "https://app.ntut.edu.tw/aptreeList.do",
     "AptreeAAListPage": "https://app.ntut.edu.tw/?apDn=ou=aa,ou=aproot,o=ldaproot",
     "SsoLoginCourseSystem": "https://app.ntut.edu.tw/ssoIndex.do?apOu=aa_0010-&apUrl=https://aps.ntut.edu.tw/course/tw/courseSID.jsp"
+  },
+  "COURESESYSTEM": {
```

Comment:
Is this right naming conventions? It is hard to identify

Reply:
And it looks like it has mixed with various naming conventions

Reply:
fix it, for some reason we pick up this shitty style in last pr...